cpp/cmake/thirdparty/patches/nvbench_override.json

```
@@ -2,13 +2,8 @@
 {
   "packages" : {
     "nvbench" : {
-      "patches" : [
-        {
-          "file" : "${current_json_dir}/nvbench_global_setup.diff",
-          "issue" : "Fix add support for global setup to initialize RMM in nvbench [https://github.com/NVIDIA/nvbench/pull/123]",
-          "fixed_in" : ""
-        }
-      ]
+      "git_url": "https://github.com/NVIDIA/nvbench.git",
+      "git_tag": "5ee8811a1ac5a90f73a4dc52ab8572c25724a0e8"
```

This tag should be updated in https://github.com/rapidsai/rapids-cmake/blob/8b1a1e0e2302ec5a6cfeed762c4f281268e7adca/rapids-cmake/cpm/versions.json#L84 rather than here.

---

@bdice what's the process for making PR's that depend on rapids-cmake changes? 

I don't suppose there's a way to make a cudf PR target the rapids-cmake PR? Or do we need to wait for the rapids-cmake changes to get merged first?

---

https://github.com/rapidsai/rapids-cmake/pull/584 is now active to merge these changes upstream.
But once merged it will break cudf benchmarks since our current patch will fail to apply.

So to ensure no breaks, we will need to merge this PR first, merge the rapids-cmake PR, and make a follow-up that removes the `nvbench_override.json` ( and related CMake call ).

---

We should use the same commit in this PR and the rapids-cmake PR:
```suggestion
      "git_tag": "555d628e9b250868c9da003e4407087ff1982e8e"
```

---

Updated the SHA. I'll leave this change active following Rob's suggestion above.

---

@robertmaynard I think we are ready to merge this so we can start the merge dance you described above.

---

Sounds great. :+1: